.gitignore

```diff
@@ -4,3 +4,4 @@ Dockerfile.goreleaser
 go.sum
 ops.asc
 exo
+.DS_Store
```

Comment:
:green_apple:

Reply:
It's also something that can go in global `.gitignore` :)